src/app/work-item/work-item-list/work-item-list-entry.component.ts

```diff
@@ -0,0 +1,153 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { Router } from "@angular/router";
+
+import { Logger } from '../../shared/logger.service';
+import { DropdownComponent } from './../../shared-component/dropdown/dropdown.component';
+import { DropdownOption } from './../../shared-component/dropdown/dropdown-option';
+import { Dialog } from '../../shared-component/dialog/dialog';
+import { DialogComponent } from '../../shared-component/dialog/dialog.component';
+import { WorkItem } from '../work-item';
+import { WorkItemService } from '../work-item.service';
+
+/*
+    Work Item List Entry Component - Displays a work item and action elements for it.
+
+    Inputs: workItem:WorkItem - the WorkItem to be displayed.
+    Events: selectEvent(WorkItemListEntryComponent) - Entry is selected.        
+            detailEvent(WorkItemListEntryComponent) - Detail view for entry is requested.
+            deleteEvent(WorkItemListEntryComponent) - Signals deletion (see note below!).
+
+    Note: all navigational events are delegated to a parent component:
+    detailEvent, selectEvent. The parent component has the obligation to manually call
+    select() on this component! Why? Because this allows the parent component to customize
+    the select behaviour (like multi-selects or xor selects).
+
+    All data events (deleteEvent only currently) are done inside this component (the service
+    is called to delete the workItem) and an event is delegated back to the parent for 
+    information purposes. The parent MUST NOT delete the workItem associated. The event is
+    intended for display purposes, like removing the entry element and reloading the list.
+*/
+
+@Component({
+  selector: 'work-item-list-entry',
+  templateUrl: '/work-item-list-entry.component.html',
+  styleUrls: ['/work-item-list-entry.component.scss'],
+})
+export class WorkItemListEntryComponent {
```

Comment:
Missed it while reworking it. Funny thing is, that it seems to be called anyway. Will fix it.